Updated to new syntax of dnx run

ktos opened the pull request:
Extended PR #1393 to every documentation file as @jasonwilliams200OK noticed in comments to bc1ba75.

Relevant annoucement is https://github.com/aspnet/Announcements/issues/52.

dnfclas commented:
Hi __@ktos__, I'm your friendly neighborhood .NET Foundation Pull Request Bot (You can call me DNFBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla2.dotnetfoundation.org.
    </span>

TTYL, DNFBOT;

ktos commented:
Sorry for not sending this all in the previous PR, it was closed too fast for me to make more changes ;-)

dnfclas commented:
__@ktos__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, DNFBOT;

adityamandaleeka commented:
Thanks for updating the other platforms as well @ktos.

akoeplinger commented:
Hmm, the change from `dnx . run` -> `dnx run` was made in [beta7](https://github.com/aspnet/Announcements/issues/52) (which isn't released yet), but the guide mentions beta5 everywhere. Should we update the docs to beta7 and mention the unstable feed?

ktos commented:
Example for `dnvm install` already installs from unstable feed: `dnvm install -r coreclr -arch x64 latest -u`, so everyone typing blindly commands (like me yesterday) will get beta7.

I could change example returns into mentioning more current unstable versions, will it be ok?

akoeplinger commented:
Yeah, good idea. The only place where it doesn't talk about -u is custom-dnx-instructions.md, so I think we should update it there as well.

ellismg commented:
/cc @blackdwarf

blackdwarf commented:
LGTM.

ktos commented:
Rebased and changed example returns to current beta7 unstable version.

jasonwilliams200OK commented:
@ktos, the rebase went wrong. Now this PR has unrelated commits from the history.

jasonwilliams200OK commented:
@ktos, you may want to see https://github.com/dotnet/coreclr/pull/1111#issuecomment-109700720.

jasonwilliams200OK commented on Documentation/install/get-dotnetcore-dnx-windows.md:
```diff
@@ -25,35 +25,35 @@ You can see the currently installed DNX versions with `dnvm list`, which will di
+This will install the 32-bit version of .NET Core. If you want the 64-bit version, you can specify processor architecture:
 
 	C:\coreclr-demo> dnvm install -r coreclr -arch x64 latest -u
 
```
It can be changed to:

\`\`\`cmd
C:\coreclr-demo> dnvm install -r coreclr -arch x64 latest -u 
\`\`\`

So it looks like:

```cmd
C:\coreclr-demo> dnvm install -r coreclr -arch x64 latest -u 
```

jasonwilliams200OK commented on Documentation/install/get-dotnetcore-dnx-windows.md:
```diff
@@ -25,35 +25,35 @@ You can see the currently installed DNX versions with `dnvm list`, which will di
+	Active Version           Runtime Architecture Location                       Alias
+	------ -------           ------- ------------ --------                       -----
+      *    1.0.0-beta7-12364 coreclr x86          C:\Users\rlander\.dnx\runtimes
+           1.0.0-beta7-12364 coreclr x64          C:\Users\rlander\.dnx\runtimes		   
```
nit: extra spaces at the end of line 56.

blackdwarf commented on Documentation/install/get-dotnetcore-dnx-linux.md:
```diff
@@ -64,8 +64,8 @@ You can see the currently installed DNX versions with `dnvm list`.
-  *    1.0.0-beta5-11649    coreclr x64  ~/.dnx/runtimes
-       1.0.0-beta5-11649    mono         ~/.dnx/runtimes      default
+  *    1.0.0-beta7-12364    coreclr x64  ~/.dnx/runtimes
+       1.0.0-beta7-12364    mono         ~/.dnx/runtimes      default
```
Would be good to add a note here saying something to the effect of "your display may vary as new versions of the DNX are published" or similar.

ktos replied:
Good idea, added such note to every first time usage of `dnvm list`.

ktos commented:
Thanks @jasonwilliams200OK, messed up a bit, fixed now (and extra characters at end of the line too). Added comment after dnvm list that display may vary with new published versions.

Why change four spaces indent into three acute syntax? Is there a difference?

jasonwilliams200OK commented:
Great!

> Why change four spaces indent into three acute syntax? Is there a difference?

If you are referring to this comment: https://github.com/dotnet/coreclr/pull/1395#discussion_r37312775, then I was suggesting to wrap code in **\`\`\`cmd** for syntax highlighting, not the indentation.
Also please consider addressing [comment by @blackdwarf]( https://github.com/dotnet/coreclr/pull/1395#discussion_r37313519).

ktos commented:
But... there is no syntax highlighting for cmd. To add more, I am not sure if those examples should have syntax highlighting, as they are not copyable (e.g. with whole command prompt).

@blackdwarf note about new versions was added: https://github.com/dotnet/coreclr/pull/1395/files#diff-7a8821bc5d6970040202f459661ed950R60

jasonwilliams200OK commented:
> there is no syntax highlighting for cmd. 

That is not true. There is syntax highlighting for \*everthing\* (over 300 languages including bash,cmd,ini,powershell and you-name-it) in [GitHub Flavored Markdown](https://help.github.com/articles/github-flavored-markdown/). In fact GitHub uses https://bitbucket.org/birkenfeld/pygments-main/ and https://github.com/github/linguist libraries.

That change I am proposing can be delivered in a separate PR.
Thanks for your effort! :+1:

ellismg commented:
Thanks for the contribution @ktos!
